packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -50,10 +50,10 @@ class _DefaultHeroTag {
 ///
 /// See also:
 ///
-///  * [Scaffold]
-///  * [RaisedButton]
-///  * [FlatButton]
-///  * <https://material.google.com/components/buttons-floating-action-button.html>
+///  * [Scaffold], in which floating action buttons typically live.
+///  * [RaisedButton], another kind of button that appears to float above the
+///    content.
```

Comment:
Nit: should FlatButton be included in this list?

Reply:
I included raised button because it floats. I think if we start including all the buttons, this will quickly be overwhelming for readers.